packages/java/parser-jvm-plugin-backbone/src/main/java/dev/hilla/parser/plugins/backbone/EndpointExposedPlugin.java

```diff
@@ -35,6 +37,10 @@ public void exit(NodePath<?> nodePath) {
     @Override
     public Node<?, ?> resolve(@Nonnull Node<?, ?> node,
             @Nonnull NodePath<?> parentPath) {
+        if (node instanceof MethodNode
+                && parentPath.getNode() instanceof EndpointExposedNode) {
```

Comment:
What if the `EndpointExposed` class is not the parent one, but it's up in the hierarchy?

Reply:
The [POC Jackson parser](https://github.com/cromoteca/jackson-parser/blob/8813055c665142f951280e2b25f31590a31de99f/generator/src/main/java/com/cromoteca/parser/Parser.java#L57-L66) I wrote when testing Jackson for the generator does a better job in my opinion about how to find methods.

Reply:
The thought also occured to me that EndpointExposed is probably not the only case. If a more generic solution is implemented, this fix can be reverted and the test kept.